Cargo.toml

```diff
@@ -24,16 +27,16 @@ prost = "0.13"
 reqwest = { version = "0.12", features = ["json"] }
 reth-chainspec = { git = "https://github.com/paradigmxyz/reth", tag = "v1.3.3", default-features = false }
 reth-primitives = { git = "https://github.com/sp1-patches/reth", tag = "rsp-20240830", default-features = false, features = [
-    "alloy-compat",
-    "optimism",
-    "std",
+  "alloy-compat",
+  "optimism",
+  "std",
 ] }
-rsp-primitives = { git = "https://github.com/succinctlabs/rsp.git", rev= "837a8c04a29deadb576a75fcf56136f26ce5dd4d" }
-rsp-client-executor = { git = "https://github.com/succinctlabs/rsp.git", rev= "837a8c04a29deadb576a75fcf56136f26ce5dd4d" }
-rsp-host-executor = { git = "https://github.com/succinctlabs/rsp.git", rev= "837a8c04a29deadb576a75fcf56136f26ce5dd4d" }
-rsp-rpc-db = { git = "https://github.com/succinctlabs/rsp.git", rev= "837a8c04a29deadb576a75fcf56136f26ce5dd4d" }
+rsp-primitives = { git = "https://github.com/succinctlabs/rsp.git", rev = "8308457c8a4ebf384b6e428d904c68ba6d8ff731" }
+rsp-client-executor = { git = "https://github.com/succinctlabs/rsp.git", rev = "8308457c8a4ebf384b6e428d904c68ba6d8ff731" }
+rsp-host-executor = { git = "https://github.com/succinctlabs/rsp.git", rev = "8308457c8a4ebf384b6e428d904c68ba6d8ff731" }
+rsp-rpc-db = { git = "https://github.com/succinctlabs/rsp.git", rev = "8308457c8a4ebf384b6e428d904c68ba6d8ff731" }
```

Comment:
this was updated to 5, uses the latest commit over `branch = "main"` used in another place.